pyproject.toml

```diff
@@ -1,6 +1,119 @@
 [build-system]
-requires = ["setuptools>=46.4.0", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.5,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "jupyter_book"
```

Comment:
The only "concern" of this name change, from `jupyter-book` to `jupyter_book`, is that although pip treats them the same (normalising `-` and `_`), conda for example does not. So I'm not sure if there would definitely be any issue.

You can keep the name as `jupyter-book`, and set the module name separately, with:

```yaml
[tool.flit.module]
name = "jupyter_book"
```

Reply:
Ah perfect, will do this.